README.md

```diff
@@ -1,10 +1,12 @@
 # Monitoring
 This repository contains examples for configuring monitoring of Redpanda using Prometheus and Grafana.
 
----
-**NOTE**: This version is designed to work with the `public_metrics` endpoint that was introduced in Redpanda 22.2 and is provided by Redpanda Cloud. If you are running an older version of Redpanda please use the [legacy version](../../tree/legacy_metrics) of these dashboards.
+> [!NOTE]
+> This version is designed to work with the `public_metrics` endpoint that was introduced in Redpanda 22.2 and is provided by Redpanda Cloud. If you are running an older version of Redpanda please use the [legacy version](../../tree/legacy_metrics) of these dashboards.
+
+> [!IMPORTANT]
+> The latest version of the [Redpanda Ops Dashboard](grafana-dashboards/Redpanda-Ops-Dashboard.json) includes the newer Kafka handler latency metric, specifically `redpanda_kafka_handler_latency_seconds_bucket{handler="produce"}` and `redpanda_kafka_handler_latency_seconds_bucket{handler="consume"}`. This metric is only available from Redpanda versions `23.2+` and `23.1.18+`.
```

Comment:
The quoted versions here are for the internal metrics. 

They are only on the public endpoint as of v23.1.19+, v23.2.12+